Source/UserManagement/Domain/StaffUser/AggregateRoots/StaffUserManagement.cs

```diff
@@ -0,0 +1,161 @@
+using System;
+using Concepts;
+using doLittle.Domain;
+using Domain.StaffUser.Commands;
+using Events.StaffUser;
+
+namespace Domain.StaffUser.AggregateRoots
+{
+    public class StaffUserManagement : AggregateRoot
+    {
+        public StaffUserManagement(Guid id) : base(id)
+        {
+        }
+        
+        public void AddStaffUser(AddStaffUser command)
```

Comment:
This should be mutiple commands.  Put the role into the command name and let the front end send the correct command based on what role is selected.  Pretty much you have the command names, they are the methods minus the Handle prefix. 

In general terms, a property that indicates a type is a code smell in DDD.

Also, commands shouldn't be passed to the AggregateRoot, the parameters should be extracted and passed into well defined methods.